packages/webcomponents/src/elements.ts

```diff
@@ -1,7 +1,7 @@
 import { Builder, builder } from '@builder.io/react';
 const importReact = () => import('@builder.io/react');
-const importShopify = () => import('@builder.io/shopify/react');
-const importShopifyJs = () => import('@builder.io/shopify/js');
+const importShopify = () => import('@builder.io/shopify/dist/react/bundle.esm.js');
+const importShopifyJs = () => import('@builder.io/shopify/dist/js');
```

Comment:
That's what the original non-dist files were re-exporting, so this should be OK:

https://github.com/BuilderIO/builder/blob/5019f07c2cc718de230713daa0e5986771220104/packages/shopify/react/index.js#L1

https://github.com/BuilderIO/builder/blob/5019f07c2cc718de230713daa0e5986771220104/packages/shopify/js.js#L1